apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -210,6 +231,45 @@ export default function ListBoxSearch({
     return undefined;
   };
 
+  const focusOnInput = () => {
+    inputRef.current?.setAttribute('tabindex', 0);
+    inputRef.current?.focus();
+    inputRef.current?.setAttribute('tabindex', -1);
```

Comment:
why two setAttribute with tabindex?

I think
```suggestion
    inputRef.current?.setAttribute('tabindex', -1);
    inputRef.current?.focus();
```
would do the same thing

Reply:
I had some issues to focus back on input, so set tabindex to 0. Seems don't need to change tabindex at all, and focus back works. I will remove change tabindex logic